src/App.js

```diff
@@ -37,7 +37,9 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      {tweetsArray.map((tweet) => {
+        return <Tweet tweet = {tweet} />;
```

Comment:
remember when we use a map, we need to pass the key prop so react knows which node refers,  in general we have and id when we are rendering data. in this case we dont have an id so we can do something in this case like this is use the name of the user ONLY for this case
```suggestion
        return <Tweet tweet = {tweet}  key={tweet.user.name}/>;
```